<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMIDialog</class>
 <widget class="QWidget" name="BMIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalkulator BMI</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLineEdit" name="weightInput">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Waga (kg)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="heightInput">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Wzrost (m)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculateButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Oblicz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wynik BMI...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
